tests/units/anta_tests/snmp/data.py

```diff
@@ -0,0 +1,146 @@
+"""Data for testing anta.tests.snmp"""
+
+from typing import Any, Dict, List
+
+INPUT_SNMP_STATUS: List[Dict[str, Any]] = [
+    {
+        "name": "success",
+        "eos_data": [
+            {
+                'vrfs': {'snmpVrfs': ['MGMT', 'default']},
+                'enabled': True
+            }
+        ],
+        "side_effect": "MGMT",
+        "expected_result": "success",
+        "expected_messages": []
+    },
+    {
+        "name": "failure",
+        "eos_data": [
+            {
+                'vrfs': {'snmpVrfs': ['default']},
+                'enabled': True
+            }
+        ],
+        "side_effect": "MGMT",
+        "expected_result": "failure",
+        "expected_messages": ["SNMP agent disabled in vrf MGMT"]
+    },
+    {
+        "name": "failure",
+        "eos_data": [
+            {
+                'vrfs': {'snmpVrfs': ['default']},
+                'enabled': False
+            }
+        ],
+        "side_effect": "default",
+        "expected_result": "failure",
+        "expected_messages": ["SNMP agent disabled in vrf default"]
+    },
+]
+
+INPUT_SNMP_IPV4_ACL: List[Dict[str, Any]] = [
```

Comment:
Add tests for skipped scenarios as well - cf https://github.com/arista-netdevops-community/anta/blob/master/tests/units/anta_tests/system/data.py#L21